server.py

```diff
@@ -197,43 +197,71 @@ def delete_inventories(inventory_id):
         inventory.delete()
     return make_response('', status.HTTP_204_NO_CONTENT)
 
+
 ######################################################################
 # COUNT TOTAL QUANTITY OF PRODUCT WITH GIVEN NAME
 ######################################################################
 @app.route('/inventories/count', methods=['GET'])
 def count_inventories_quantity():
-	# return a list of Inventory
+    # return a list of Inventory
     name = request.args.get('name')
     if name:
         find_by_id_records = Inventory.find_by_name(name)
-        quantities = [ record.quantity for record in find_by_id_records]
+        quantities = [record.quantity for record in find_by_id_records]
         quantity_sum = sum(quantities)
-        message = {'count' : quantity_sum}
-    return make_response(jsonify(message),  status.HTTP_200_OK )
+        message = {'count': quantity_sum}
+    return make_response(jsonify(message), status.HTTP_200_OK)
 
 
 ######################################################################
 # QUERY INVENTORIES
 ######################################################################
-@app.route('/inventories/query', methods = ['GET'])
+@app.route('/inventories/query', methods=['GET'])
 def query_inventories_by_name_status():
-    name = request.args.get('name').strip()
-    status = request.args.get('status').strip()
-   
```

Comment:
strip() is wrong? I was thinking to remove redundant spaces.

Reply:
No, strip() is not wrong. I just think that strip() is not necessary for now, so I removed it.

Reply:
The problem is that I need to serialize the inventory before using the [x for x in A if x in B] statement.

```
results1 = [inventory.serialize() for inventory in inventories_by_name]
results2 = [inventory.serialize() for inventory in inventories_by_status]
results = [r for r in results1 if r in results2]
```